standards/core/sync.md

```diff
@@ -7,126 +7,186 @@ contributors:
  - Hanno Cornelius <hanno@status.im>
 ---
 
-## Abstract
-This specification explains the `WAKU-SYNC` protocol
-which enables the reconciliation of two sets of message hashes
-in the context of keeping multiple Store nodes synchronized.
-Waku Sync is a wrapper around
-[Negentropy](https://github.com/hoytech/negentropy) a [range-based set reconciliation protocol](https://logperiodic.com/rbsr.html).
+# Abstract
+This specification explains `WAKU-SYNC`
+which enables the synchronization of messages between 2 Store nodes.
 
-## Specification
+# Specification
 
-**Protocol identifier**: `/vac/waku/sync/1.0.0`
+Waku Sync consists of 2 protocols; reconciliation and transfer.
+Reconciliation is the process of finding differences in 2 sets of message hashes.
+Transfer is then used to bilaterally send messages to the other peer.
+The end goal being that both peers have the same set of hashes and messages.
 
-### Terminology
+#### Terminology
 The key words “MUST”, “MUST NOT”, “REQUIRED”, “SHALL”, “SHALL NOT”, “SHOULD”, “SHOULD NOT”, 
 “RECOMMENDED”, “MAY”, and “OPTIONAL” in this document are to be interpreted as described in [RFC2119](https://www.ietf.org/rfc/rfc2119.txt).
 
-The term Negentropy refers to the protocol of the same name.
-Negentropy payload refers to
-the messages created by the Negentropy protocol.
-Client always refers to the initiator
-and the server the receiver of the first payload.
-
-### Design Requirements
-Nodes enabling Waku Sync SHOULD
-manage and keep message hashes in a local cache
-for the range of time
-during which synchronization is required.
-Nodes SHOULD use the same time range,
-for Waku we chose one hour as the global default.
-Waku Relay or Light Push protocol MAY be enabled
-and used in conjunction with Sync
-as a source of new message hashes
-for the cache.
-
-Nodes MAY use the Store protocol
-to request missing messages once reconciliation is complete
-or to provide messages to requesting clients.
-
-### Payload
+## Reconciliation
 
+**Libp2p Protocol identifier**: `/vac/waku/reconciliation/1.0.0`
+
+#### Message Ids
+Message Ids MUST be composed of the timestamp and the hash of the Waku messages.
+
+The timestamp MUST be the time of creation and
+the hash MUST follow the
+[deterministic message hashing specification](https://rfc.vac.dev/waku/standards/core/14/message#deterministic-message-hashing)
+
+> This way the message Ids can always be totally ordered.
+Chronologically according to the timestamp and
```

Comment:
The message IDs are ordered chronologically,....

(Sentence needs a subject and verb)